Assignment 5/SenerioOne.js

```diff
@@ -0,0 +1,34 @@
+try 
+{
+  function CarAccidentByWoman() 
+  {
+    console.log("Women did accident of her Car");
+  }
```

Comment:
please follow the same pattern of formatting rules in each of the files, as the indentation is SenerioOne.js has lesser horizontal indentation from left space as compared to SenerioTwo.js